### ISSUE TITLE
filedefs never loaded from User folder on Win10

### ISSUE THREAD

**Issue by grubernd** [geany/geany#2447]
Geany 1.36

the filedefs from the user folder never get loaded.

tested this..
open: Tools > Configuration Files > filetypes.common
this file is stored in `Users\USER\AppData\Roaming\geany\filedefs`
all lines start with `#~`
file can be edited and saved, but no change in the UI, neither through reload nor restart.

when one digs into the application folder at
`Program Files (x86)\Geany\data\filedefs`
and changes file permissions so a user can edit them,
then any change gets picked up by the program.

**Comment by codebrainz** [comment]
You removed the `#~` comments right? (<kbd>Ctrl</kbd>+<kbd>a</kbd> then <kbd>Ctrl</kbd>+<kbd>e</kbd>)

**Comment by grubernd** [comment]
yes, of those items that I wanted to have changed.

**Comment by elextr** [comment]
please post the contents of `help->debug messages`

**Comment by grubernd** [comment]
this is everything until the first file would get loaded.
```
00:03:52: Geany INFO		: Geany 1.36, German_Germany.1252
00:03:52: Geany INFO		: GTK 2.24.32, GLib 2.60.6
00:03:52: Geany INFO		: System data dir: C:\Program Files (x86)\Geany\data
00:03:52: Geany INFO		: User config dir: C:\Users\michael\AppData\Roaming\geany
00:03:52: Geany INFO		: System plugin path: C:\Program Files (x86)\Geany\lib\geany
00:03:52: Geany INFO		: Added filetype Arduino (61).
00:03:52: Geany INFO		: Added filetype Clojure (62).
00:03:52: Geany INFO		: Added filetype CUDA (63).
00:03:52: Geany INFO		: Added filetype Cython (64).
00:03:52: Geany INFO		: Added filetype Genie (65).
00:03:52: Geany INFO		: Added filetype Graphviz (66).
00:03:52: Geany INFO		: Added filetype Groovy (67).
00:03:52: Geany INFO		: Added filetype JSON (68).
00:03:52: Geany INFO		: Added filetype Kotlin (69).
00:03:52: Geany INFO		: Added filetype Nim (70).
00:03:52: Geany INFO		: Added filetype Scala (71).
00:03:52: Geany INFO		: Added filetype Swift (72).
00:03:52: Geany INFO		: Added filetype TypeScript (73).
00:03:52: GLib-GIO DEBUG	: _g_io_module_get_default: Found default implementation winhttp (GWinHttpVfs) for ‘gio-vfs’
```

**Comment by elextr** [comment]
Well, the `User config dir` shown there does not match where you posted the file is stored, so thats probably the problem.

**Comment by grubernd** [comment]
i posted the respective actual folders inside the base folders.
and even if the folder would be wrong (which i doubt) the GUI should open the correct file. as of now - it opens the intended correct file but that file never gets used. btw, same thing works fine on my linux machines.

**Comment by elextr** [comment]
BTW did you restart or `tools->reload configuration` after editing?

**Comment by grubernd** [comment]
both. multiple times. in various combinations. with different settings applied, commented and uncommented.
(for reference: worked ten years as beta-tester)

**Comment by elextr** [comment]
Oh you already said that, ok, so next question, what did you edit?

**Comment by elextr** [comment]
(Sorry for the obvious but it works sooooo many times :)

**Comment by grubernd** [comment]
my tests were centered around
`current_line=`
true/false and changing colors

**Comment by elextr** [comment]
another obvious one, you do have the colour scheme set on default?

**Comment by grubernd** [comment]
yes. i have installed from on a machine that never before hat geany.
also it was the first thing i tried to change.

**Comment by elextr** [comment]
Ok, well, since I don't have windows, and you said it works for you on linux, then I'm outa ideas, somebody else?

**Comment by codebrainz** [comment]
More of a work-around, but you could try to create/edit a colour scheme in your user directory, instead of changing `filetypes.common`. IIRC the Win32 installer comes with the option to install [geany-themes](https://github.com/geany/geany-themes) which includes a bunch themes, one of which you could use as a base for your own.

**Comment by eht16** [comment]
@grubernd just tested on Windows: works for me.
I've got `C:\Users\enrico\AppData\Roaming\geany\filedefs\filetypes.c` with the following content:
```ini
[styling]
default=default
comment=#FF5F5F
commentline=#FF5F5F
commentdoc=#FF5F5F
preprocessorcomment=#FF5F5F
preprocessorcommentdoc=#FF5F5F

[build-menu]
EX_00_LB=_Execute
EX_00_CM="./%e"
EX_00_WD=
FT_00_LB=_Compile
FT_00_CM=C:\\msys64\\mingw32\\bin\\i686-w64-mingw32-gcc.exe -Wall -c "%f"
FT_00_WD=
FT_01_LB=_Build
FT_01_CM=C:\\msys64\\mingw32\\bin\\i686-w64-mingw32-gcc.exe -Wall -o "%e" "%f"
FT_01_WD=
```
(the styling settings are just for testing, the build settings are productive)

Which exact settings did you try to modify? Maybe it helps if you share the file?
